CMakeLists.txt

```diff
@@ -23,9 +23,8 @@ find_package(std_msgs REQUIRED)
 find_package(geometry_msgs REQUIRED)
 find_package(laser_geometry REQUIRED)
 find_package(tf2 REQUIRED)
-# uncomment the following section in order to fill in
-# further dependencies manually.
-# find_package(<dependency> REQUIRED)
+# fmt is available as a rosdep
+find_package(fmt REQUIRED)
```

Comment:
Since fmt is not being used right now, could you remove it from the CMakeLists and package.xml? Other than that, looks good to me:)